### ISSUE TITLE
Added "[x.x]" subchapters support

### ISSUE THREAD

**Issue by YuuichiLV** [oae/kaizoku#103]
### Feature Description

When using MangaDex, haven't tried others yet, if there is a sub chapter say "20.5" Kaizoku will save it as "21" which messes up the chapter order.

### Solution you would like

Please enable decimals in the chapter naming. Example : "[0001]" to "[0001.0]".

**Comment by eswiig** [comment]
Yes please

**Comment by otomay** [comment]
plsss

**Comment by YuuichiLV** [comment]
Is this project abandon? No updates from February.

**Comment by Elryyyyyyyy** [comment]
Yeah seems like the dev ditched their project without any notice, hate such guys.
Gonna try and see if i can fix it myself, the metadata is also wrong.

**Comment by oae** [comment]
> Yeah seems like the dev ditched their project without any notice, hate such guys.
> Gonna try and see if i can fix it myself, the metadata is also wrong.

As my status shows on GitHub i am on vacation. I don't want anyone in my repo that doesn't have any respect to other people

**Comment by ElryWeeb** [comment]
> > Yeah seems like the dev ditched their project without any notice, hate such guys.
> > Gonna try and see if i can fix it myself, the metadata is also wrong.
> 
> As my status shows on GitHub i am on vacation. I don't want anyone in my repo that doesn't show any respect to other people

On Vacation, mate the last update was on march, who are you trying to fool.
You banning my other account wont fix or excuse your lazyness.
Im respecting other people and i would respect you, if you had a valid reason to not update this project anymore.

**Comment by MugenEnzan** [comment]
> > > Yeah seems like the dev ditched their project without any notice, hate such guys.
> 
> > > Gonna try and see if i can fix it myself, the metadata is also wrong.
> 
> > 
> 
> > As my status shows on GitHub i am on vacation. I don't want anyone in my repo that doesn't show any respect to other people
> 
> 
> 
> On Vacation, mate the last update was on march and your vacation badge is there for so long...
> 
> People just want to use your project, but you have abandoned it since march...
> Just please update and fix Kaizoku.

Just be grateful someone is working on this. Something that many people want. 

I understand if you're frustrated but at least be respectful and grateful. 

Basic human decency goes a long way.

**Comment by Captainpax** [comment]
There's someone working on this? Where?

**Comment by ElryWeeb** [comment]
> > > > Yeah seems like the dev ditched their project without any notice, hate such guys.
> > 
> > 
> > > > Gonna try and see if i can fix it myself, the metadata is also wrong.
> > 
> > 
> > > 
> > 
> > 
> > > As my status shows on GitHub i am on vacation. I don't want anyone in my repo that doesn't show any respect to other people
> > 
> > 
> > On Vacation, mate the last update was on march and your vacation badge is there for so long...
> > People just want to use your project, but you have abandoned it since march...
> > Just please update and fix Kaizoku.
> 
> Just be grateful someone is working on this. Something that many people want.
> 
> I understand if you're frustrated but at least be respectful and grateful.
> 
> Basic human decency goes a long way.

There is noone working on this, there is a difference between human decency and realism.

**Comment by Ambushfall** [comment]
> > > > > Yeah seems like the dev ditched their project without any notice, hate such guys.
> > > 
> > > 
> > > > > Gonna try and see if i can fix it myself, the metadata is also wrong.
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > As my status shows on GitHub i am on vacation. I don't want anyone in my repo that doesn't show any respect to other people
> > > 
> > > 
> > > On Vacation, mate the last update was on march and your vacation badge is there for so long...
> > > People just want to use your project, but you have abandoned it since march...
> > > Just please update and fix Kaizoku.
> > 
> > 
> > Just be grateful someone is working on this. Something that many people want.
> > I understand if you're frustrated but at least be respectful and grateful.
> > Basic human decency goes a long way.
> 
> There is noone working on this, there is a difference between human decency and realism.

People, be respectful of others, if there's something not working, do not request it, find a way to solve it, then post the solution if you want. Or just fix it for yourself and move on. Such is the nature of open source that not everyone will maintain everything. 
Cheers

**Comment by otomay** [comment]
@ElryWeeb did you stop working on it?

**Comment by ElryWeeb** [comment]
@otomay Yes the code is an absolute mess
